### PR TITLE
Adding some analogy for better explanation of data pages

### PR DESCRIPTION
**Pull request by PiJoCoder**

### CODE REVIEW

**Comment by PRMerger16** [comment]
@PiJoCoder : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MikeRayMSFT** [comment]
@PiJoCoder  - reviewing.

File: docs/relational-databases/pages-and-extents-architecture-guide.md [hunk 1]
```diff
@@ -24,11 +24,13 @@ The page is the fundamental unit of data storage in [!INCLUDE[ssNoVersion](../in
 
 The fundamental unit of data storage in [!INCLUDE[ssNoVersion](../includes/ssnoversion-md.md)] is the page. The disk space allocated to a data file (.mdf or .ndf) in a database is logically divided into pages numbered contiguously from 0 to n. Disk I/O operations are performed at the page level. That is, SQL Server reads or writes whole data pages.
 
-Extents are a collection of eight physically contiguous pages and are used to efficiently manage the pages. All pages are stored in extents.
+Extents are a collection of eight physically contiguous pages and are used to efficiently manage the pages. All pages are tracked in extents.
```
Comment (bluefooted) [inline docs/relational-databases/pages-and-extents-architecture-guide.md]: Change "tracked in" to "organized into"

**Comment by PRMerger17** [comment]
@PiJoCoder : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by rothja** [comment]
@PiJoCoder Thanks, again for suggesting the analogy. We like it! I'm going to merge these in and the changes will appear soon. Thank you! + @MikeRayMSFT @bluefooted #sign-off

**Pull request merged**